ubg/src/pages/Home.js

```diff
@@ -1,9 +1,168 @@
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import IconButton from '@material-ui/core/IconButton';
+import FavoriteIcon from '@material-ui/icons/Favorite';
+import Typography from '@material-ui/core/Typography';
+import Star from '@material-ui/icons/Star';
+import Timer from '@material-ui/icons/Timer';
+import People from '@material-ui/icons/People';
+import Grid from '@material-ui/core/Grid';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import MenuIcon from '@material-ui/icons/Menu';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  menuButton: {
+    marginRight: theme.spacing(2),
+  },
+  title: {
+    flexGrow: 1,
+  },
+  card: {
+    maxWidth: 390,
+  },
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 120,
+  },
+  selectEmpty: {
+    marginTop: theme.spacing(2),
+  }
+}));
 
 export default function Home() {
-    return (
-        <div>
-            home
-        </div>
-    )
+  const [minAge, setMinAge] = React.useState('1');
+  const [maxAge, setMaxAge] = React.useState('Max');
+  const [minPlayer, setMinPlayer] = React.useState('1');
+  const [maxPlayer, setMaxPlayer] = React.useState('8');
+  const [minTime, setMinTime] = React.useState('5min');
+  const [maxTime, setMaxTime] = React.useState('Max');
+  return (
+    <div>
+      <Navbar/>
+      <Box  boxShadow={1} m={10}>
+        <Filter label = "Minimum Age" value={minAge} menu={[5, 10, 16, 20, 35]} max={false} onChange={(v) => setMinAge(v)} />
+        <Filter label = "Maximum Age" value={maxAge} menu={[5, 10, 16, 20, 35]} max={true} onChange={(v) => setMaxAge(v)} />
+        <Filter label = "Minimum Player" value={minPlayer} menu={[2, 3, 4, 5, 6, 7, 8]} max={false} onChange={(v) => setMinPlayer(v)}/>
+        <Filter label = "Maximum Player" value={maxPlayer} menu={[1, 2, 3, 4, 5, 6, 7]} max={true} onChange={(v) => setMaxPlayer(v)} />
+        <Filter label = "Minimum Time" value={minTime} menu={["15min", "30min", "1h", "1h30min", "2h"]} max={false} onChange={(v) => setMinTime(v)} />
+        <Filter label = "Maximum Time" value={maxTime} menu={["5min", "15min", "30min", "1h", "1h30min", "2h"]} max={true} onChange={(v) => setMaxTime(v)} />
+      </Box>
+      <Box m={10}>
+        <Grid container justify="center" alignItems="center" spacing={7}>
+          <Grid item>
+            <ImgMediaCard />
+          </Grid>
+          <Grid item >
+            <ImgMediaCard />
+          </Grid>
+          <Grid item >
+            <ImgMediaCard />
+          </Grid>
+          <Grid item >
+            <ImgMediaCard />
+          </Grid>
+          <Grid item >
+            <ImgMediaCard />
+          </Grid>
+          <Grid item >
+            <ImgMediaCard />
+          </Grid>
+        </Grid>
+      </Box>
+    </div>
+  );
+
+}
+
+function Navbar() {
```

Comment:
I think we should separate the different components into different files, the same goes for Filter and ImgMediaCard.